src/main/scala/rules/Evaluator.scala

```diff
@@ -1404,7 +1404,7 @@ object evaluator extends EvaluationRules {
         for (e <- remainingTriggerExpressions)
           v.reporter.report(WarningsDuringVerification(Seq(
             VerifierWarning(s"Might not be able to use trigger $e, since it is not evaluated while evaluating the body of the quantifier", e.pos))))
```

Comment:
Is this message still accurate? Instead of "Might not be able to use trigger $e", don't we know that they will not be used?

Reply:
The reason it says "might" is because we know it can't use the trigger *on the current branch*, but it might be able to on different branches. Could be more explicit about that though I guess.